python/nav/models/oui.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from nav.models.fields import VarcharField
+
+
+class OUI(models.Model):
+    """Defines an OUI and the name of the vendor the OUI belongs to"""
+
+    vendor = VarcharField()
+    oui = models.CharField(max_length=17, unique=True)
```

Comment:
Actually, after looking at #2945, I'm reconsidering this.  I think this model should be using oui as its primary key and drop the "standard" serial primary key, which seems entirely pointless to me now...

Reply:
diddly done